db/migrate/20210421045714_create_log_times.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLogTimes < ActiveRecord::Migration[6.1]
+  def change
+    create_table :log_times do |t|
```

Comment:
wouldn't you need to add the corresponding id's for those realtions? e.g. branches_id, user_id.